### ISSUE TITLE
MalmoEnv basic tutorial crash

### ISSUE THREAD

**Issue by trigaten** [microsoft/malmo#916]
I am attempting to use MalmoEnv on Mac
I ran `./launchClient.sh -port 9000 -env` which works and launches the client.
Then I ran `python3 run.py --mission missions/mobchase_single_agent.xml --port 9000 --episodes 10` in the appropriate directory. This crashed the client with this error:

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':runClient'.
> Process 'command '/Library/Java/JavaVirtualMachines/temurin-8.jdk/Contents/Home/bin/java'' finished with non-zero exit value 134

The terminal in which I ran the mission comment showed:

reset 0
Pause before retry on a bytes-like object is required, not 'NoneType'

Any ideas?

**Comment by rainingDesert** [comment]
I also got this error on Mac. May I ask if you solved the problem after that?

**Comment by trigaten** [comment]
> I also got this error on Mac. May I ask if you solved the problem after that?

I cant remember if I eventually fixed this. However, Malmo is no longer supported. Use https://minerl.readthedocs.io/en/latest/

**Comment by rainingDesert** [comment]
> 

Thanks for your reply. I went through the Github of minerl. But I need to use several environments provided in Malmo, which seem not to be provided in minerl. Do you know any documents of building custom environments with minerl?

**Comment by trigaten** [comment]
Yep, I wrote them! See here: https://minerl.readthedocs.io/en/v0.4.4/tutorials/custom_environments.html

Note that this is not for the most up to date version of the codebase. However, most of the code should work if you choose to use version 1.0.0

**Comment by rainingDesert** [comment]
Seems interesting, thanks so much.

By the way, can minerl load .xml or other formats file like Malmo?

**Comment by trigaten** [comment]
I believe so, but you may have to modify the source a bit to do that.
